Automattic/jetpack
# VideoPress: Scaffolding Video Chapters block

Closes #27050 

#### Changes proposed in this Pull Request:

* Add new `video-chapters` folder.
* Include it as beta from `Jetpack` plugin itself.
* Insert new config at `webpack.config` to build `video-chapters` block.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






- Set `JETPACK_BETA_BLOCKS` as `true` in your `wp-config.php` file.

_VideoPress_

* Activate only `VideoPress` plugin.
* Start a new post.
* You should be able to insert `Video Chapters` block.

_Jetpack_

* Activate only `Jetpack` plugin.
* Start a new post.
* You should be able to insert `Video Chapters (beta)` block.
